source/mirror.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Mirrors
+---
+# Mirrors
+
+You can download the latest distribution files from the following FTP
+areas:
+
+| Locale | URL                                                |
+|--------|----------------------------------------------------|
+| CZ     | <ftp://ftp.fi.muni.cz/pub/openssl/>                |
+| DE     | <ftp://ftp.pca.dfn.de/pub/tools/net/openssl/>      |
+| DE     | <http://artfiles.org/openssl.org/>                 |
+| HR     | <ftp://ftp.linux.hr/pub/openssl/>                  |
+| HU     | <ftp://ftp.kfki.hu/pub/packages/security/openssl/> |
+| PL     | <ftp://guest.kuria.katowice.pl/pub/openssl/>       |
+| PL     | <rsync://ftp.tpnet.pl/pub/security/openssl/>       |
+
```

Comment:
For some reason on beta.openssl.org the table is merged with the paragraph after it. Perhaps the md->html conversion should handle this?

Reply:
Also the table styling has some glitches.

Reply:
There's a lot of weirdness happening with tables...  it's not in the generation, that looks perfectly fine, so it's something else.  In the HTML, this was solved with an added `<p>&nbsp;</p>` after the table, which is indeed an ugly workaround...

Re the rest of the glitches, I'll adjust that tomorrow

Reply:
It seems that surrounding any table with \<p>...\</p> fixes the problem

Reply:
It was resolved for the downloads table but not for the table with the list of the mirrors. Please look at https://beta.openssl.org/source/mirror.html

Reply:
Fixed